src/darwin/Framework/Matter.xcodeproj/project.pbxproj

```diff
@@ -300,6 +300,10 @@
 		7567A2102DA5ABCF00D91529 /* DefaultServerCluster.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7567A20B2DA5ABCE00D91529 /* DefaultServerCluster.cpp */; };
 		7567A2112DA5ABCF00D91529 /* ServerClusterInterface.h in Headers */ = {isa = PBXBuildFile; fileRef = 7567A20C2DA5ABCE00D91529 /* ServerClusterInterface.h */; };
 		7567A2122DA5ABCF00D91529 /* ServerClusterInterface.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7567A20D2DA5ABCE00D91529 /* ServerClusterInterface.cpp */; };
+		7567A2152E05FCD200D91529 /* EmberAttributeStorageImpl.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7567A2132E05FCD100D91529 /* EmberAttributeStorageImpl.cpp */; };
+		7567A2162E05FCD200D91529 /* EmberAttributeStorageImpl.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 7567A2132E05FCD100D91529 /* EmberAttributeStorageImpl.cpp */; };
+		7567A2172E05FCD200D91529 /* EmberAttributeStorageImpl.h in Headers */ = {isa = PBXBuildFile; fileRef = 7567A2142E05FCD100D91529 /* EmberAttributeStorageImpl.h */; };
+		7567A2182E05FCD200D91529 /* EmberAttributeStorageImpl.h in Headers */ = {isa = PBXBuildFile; fileRef = 7567A2142E05FCD100D91529 /* EmberAttributeStorageImpl.h */; };
```

Comment:
Why are both of these duplicated?

Reply:
I used the UI to add these ... I do not know, it seems very odd to me too.